cudax/include/cuda/experimental/__hierarchy/level_dimensions.cuh

```diff
@@ -38,8 +38,16 @@ struct extents_corrected : public ::cuda::std::extents<T, Extents...> {
 };
 */
 
+struct base_dimensions_handler
+{
+  // TODO: Should these two be merged into one bool?
+  static constexpr bool is_type_supported = true;
+  template <typename Level>
+  static constexpr bool is_level_supported = true;
+};
+
 template <typename Dims>
-struct dimensions_handler
+struct dimensions_handler : public base_dimensions_handler
```

Comment:
Remark: `struct` base classes are `public` by default.
```suggestion
struct dimensions_handler : base_dimensions_handler
```
I don't know why this triggers me enough to write a comment. Feel free to ignore!

Reply:
We tend to prefer explicitness about these things. Makes the code more accessible to less experienced contributors.